### PR TITLE
Adds support for legacy tests on bacon

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
* We recently updated chrome driver to 2.46 as travis chrome version was upgraded
* This is causing our PDV tests to fail on bacon 
* Added scripts to take care of legacy chrome driver for bacon

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/okta-vue/test/e2e/harness/scripts/downloadChromeDriver.sh [hunk 1]
```diff
@@ -1,18 +1,25 @@
 # Install ChromeDriver.
 OS="`uname`"
 
+if [ $# -eq 0 ]
```
Comment (robertjd) [inline packages/okta-vue/test/e2e/harness/scripts/downloadChromeDriver.sh]: I'd feel better if we just tested for eq "legacy", then this code is more self-documenting

Comment (aarongranick-okta) [reply packages/okta-vue/test/e2e/harness/scripts/downloadChromeDriver.sh]: Maybe even better, can pass in the version.  `./downloadChromeDriver 2.45`

**Comment by aarongranick-okta** [comment]
Instead of duplicating tasks, it would be nice if we could run the same tasks with different chrome versions. What if we want to test with the next version? Are we going to add `test:e2e:next` and `pretest:e2e:next` ? Ideally this could all be controlled by an environment variable.

**Comment by vijetmahabaleshwar-okta** [comment]
@aarongranick-okta - Even I don't like duplicating scripts but this solution makes it a bit simpler, I feel. Passing arguments to scripts would require more changes. Also, we would need only 2 versions of chrome driver - the latest, and legacy one for chrome. 
@wayneearl-okta is also working on script that automatically downloads the right chrome driver depending on the version of chrome. Once that is merged (at a later point), this will be much cleaner. For now, this is a quick fix to get our critical PDV tests passing.

**Pull request closed without merging**